lib/PhasarLLVM/ControlFlow/Resolver/RTAResolver.cpp

```diff
@@ -17,29 +17,30 @@
 #include "phasar/PhasarLLVM/ControlFlow/Resolver/RTAResolver.h"
 
 #include "phasar/PhasarLLVM/DB/LLVMProjectIRDB.h"
-#include "phasar/PhasarLLVM/TypeHierarchy/LLVMTypeHierarchy.h"
+#include "phasar/PhasarLLVM/TypeHierarchy/DIBasedTypeHierarchy.h"
+#include "phasar/PhasarLLVM/Utils/LLVMIRToSrc.h"
```

Comment:
remove

Reply:
Does your comment refer only to the LLVMIRToSrc.h include? The code doesn't run without the inclusion of
#include "phasar/PhasarLLVM/TypeHierarchy/DIBasedTypeHierarchy.h"

Reply:
yes